server/src/index.ts

```diff
@@ -43,7 +36,7 @@ const server = new ApolloServer({
     }
   })
 
-  server.listen().then(({ url }) => {
+  server.listen().then(({ url } : {url:string}) => {
```

Comment:
Isn't the type already known here? You use the resolve value of `server.listen()` so I guess it should be correct without the extra type def too. If not, then we might need to check TypeScript support of Apollo Server.